### DISCUSSION TITLE
Thumbnails with React

### DISCUSSION THREAD

**Discussion by JulienSacher** [nolimits4web/swiper#4362] [Q&A]
Hi guys ! I need you about SwiperJS.
In the documentation of React, there is an example of code that I can use to make thumbnails into my slider but I don't understand anything and, when I use the same code example, nothing is displaying on the screen.
Here is my code : 

```
function Slides() {
const [thumbsSwiper, setThumbsSwiper] = useState(null)
return (
<Swiper direction="horizontal" autoplay={{delay: 2500}} centeredSlides={true} className="back__slide text-center" thumbs={{ swiper: thumbsSwiper }} onSwiper={setThumbsSwiper} watchSlidesVisibility={true} watchSlidesProgress={true}>
                <SwiperSlide><img src={p1} width="80%"/></SwiperSlide>
                <SwiperSlide><img src={p2} width="80%"/></SwiperSlide>
                <SwiperSlide><img src={p3} width="80%"/></SwiperSlide>
              </Swiper>
)
}
export default Slides;
```

Thanks for your time and your help :dancing_men:

**Comment by vltansky** [comment]
take a look at this: https://www.youtube.com/watch?v=l4kFO6VQPfA&feature=emb_title

**Reply by JulienSacher** [reply]
Thanks but it didn't help me, I want to do like the following link: https://swiperjs.com/demos#thumbs-gallery, the video that you sent me have thumbs but it's just a second slider bellow the main slider who can't display the image on click to the main slider.
